compiler/noirc_errors/src/position.rs

```diff
@@ -105,20 +161,22 @@ impl From<Span> for Range<usize> {
 
 impl From<Range<u32>> for Span {
     fn from(Range { start, end }: Range<u32>) -> Self {
-        Self(ByteSpan::new(start, end))
+        Self(ByteSpan::new(start, end), SrcId::default())
```

Comment:
The `SrcId::default()` here is definitely suspicious, and likely incorrect. I think we can't have a `From<Range<u32>>` impl anymore if we need a SrcId as well.

Reply:
`From<Range<u32>>` moved to fmt module